kinder/pkg/kubeadm/config.go

```diff
@@ -66,14 +60,12 @@ func Config(kubeadmVersion *K8sVersion.Version, data ConfigData) (config string,
 	return buff.String(), nil
 }
 
-// getKubeadmConfigVersion returns the kubeadm config version corresponding to a Kubernetes kubeadmVersion
-func getKubeadmConfigVersion(kubeadmVersion *K8sVersion.Version) (string, error) {
-	// returns the corresponding config version
-	// nb v1alpha1 (that is Kubernetes v1.10.0) is out of support,
-	// v1alpha2 (that is Kuberntes v1.11.0) is out of support.
-	// v1alpha3 (that is Kuberntes v1.13.0) is out of support.
-
-	return "v1beta2", nil
+// GetKubeadmConfigVersion returns the kubeadm config version corresponding to a Kubernetes kubeadmVersion
+func GetKubeadmConfigVersion(kubeadmVersion *K8sVersion.Version) string {
+	if kubeadmVersion.AtLeast(K8sVersion.MustParseSemantic("v1.22.0")) {
```

Comment:
@neolit123 @fabriziopandini 
I'm not sure what to do here, given the fact that `Kinder` doesn't currently support v1beta3.

Reply:
i think i agree that we have to add the v1beta3 support first before this PR.

i see you have:
https://github.com/kubernetes/kubeadm/pull/2488

xref https://github.com/kubernetes/kubeadm/issues/2428